db/product.go

```diff
@@ -0,0 +1,313 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	//"database/sql"
+	logger "github.com/sirupsen/logrus"
+)
+
+const (
+	getProductCount = `SELECT count(id) from Products ;`
+	getProductQuery = `SELECT id FROM products LIMIT $1  OFFSET  ($2 -1) * $1;`
```

Comment:
can we do offset calculation at code and db query just have number like limit

Reply:
Yes Change Done. Now calculation of offset is done in Program itself than sending parameters as is to db manager.